clang.sh

```diff
@@ -1,7 +1,7 @@
 package: Clang
-version: "7.0.0"
-tag: v7.0.0-alice1
-source: https://github.com/alisw/clang
+version: "v7.1.0"
+tag: "llvmorg-7.1.0"
+source: https://github.com/llvm/llvm-project
```

Comment:
Is this providing the clang tools and extra tools as well? If not, it's likely breaking the codechecker.

Reply:
By default it should build all projects, including clang and its tools. Otherwise you can explicitly set `-DLLVM_ENABLE_PROJECTS="clang;clang-tools-extra"`